### DISCUSSION TITLE
How to add openID configuration in strapi

### DISCUSSION THREAD

**Discussion by rvigneshw** [strapi/strapi#6250] [Legacy Topics]
Adding an  openID configuration in strapi would lead to easy integration with many apps

**Comment by derrickmehaffy** [comment]
@Aurelsicoko

**Comment by Aurelsicoko** [comment]
We are going to use Passport.js to authenticate the administrators. We plan to release it with the RBAC feature in July. Could you share your insights and upvote it on our public roadmap?

https://portal.productboard.com/strapi/1-public-roadmap/c/61-sso-third-party-authentication-providers
